packages/generator/src/helper.ts

```diff
@@ -130,14 +134,7 @@ export const getEmbeddedPagesAndEmbedPdfBoxes = async (arg: {
   return { embeddedPages, embedPdfBoxes };
 };
 
-const mm2pt = (mm: number): number => {
-  // https://www.ddc.co.jp/words/archives/20090701114500.html
-  const ptRatio = 2.8346;
-
-  return parseFloat(String(mm)) * ptRatio;
-};
-
-const getSchemaSizeAndRotate = (schema: Schema) => {
+const convertSchemaDimensionsToPt = (schema: Schema) => {
```

Comment:
renamed this function to clarify that it's doing a unit conversion (it took me a lot of time to notice this!)

Reply:
Sorry...